pkg/blockerslack/reporters/blockers/blockers_reporter.go

```diff
@@ -167,6 +168,10 @@ func (tr triageResult) getTeamMessages() []string {
 	href = fmt.Sprintf("%d Untriaged Bugs", triageCount)
 	triageMsg := makeBugzillaLink(href, tr.needTriageIDs)
 
+	newCount := len(tr.newIDs)
+	href = fmt.Sprintf("%d Bugs in \"NEW\"", newCount)
+	newMsg := makeBugzillaLink(href, tr.newIDs)
```

Comment:
I expect this will overflow Slack's message limit (#14) for teams with lots of NEW bugs in the backlog, and I would prefer just post a count without including a bug-ID-based link.

Reply:
Fixed in https://github.com/openshift/bugzilla-tools/commit/985d7fe09c074a07c3e4390f6248e549128d6253